### ISSUE TITLE
Feature request: words until cursor

### ISSUE THREAD

**Issue by olafklingt** [OleMchls/atom-wordcount#116]
It would be great to have an option to list the number of words until the cursor position.

**Comment by OleMchls** [comment]
I can imagine this feature, by pressing an option key (e.g. alt) that'd update the display for the time the key is pressed. If you or anyone would like to pick this up, that would be cool!

**Comment by olafklingt** [comment]
I don't clearly understand which behaviour you suggest. I think you want to avoid another option in the plug in settings, right?

**Comment by OleMchls** [comment]
> I don't clearly understand which behaviour you suggest. I think you want to avoid another option in the plug in settings, right?

Exactly. I think this is valuable information, but can be achieved by without a settings flag, because it's something you want to see just every once in a while, and when you need it you don't want to switch to the settings. So what about only displaying this when a certain option key is defined?
